src/OpenTelemetry.Extensions.Enrichment/OpenTelemetry.Extensions.Enrichment.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetFrameworkMinimumSupportedVersion);$(NetMinimumSupportedVersion);$(NetStandardMinimumSupportedVersion)</TargetFrameworks>
```

Comment:
any reason why we need netminimumsupportedversion?

Reply:
That's the minimum .NET (core) version we support. As I understood, @CodeBlanch asked to add it [here ](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/949#discussion_r1101919242).

Reply:
`$(NetFrameworkMinimumSupportedVersion);$(NetStandardMinimumSupportedVersion)` should cover all we need. Unless we need an api only available in .net6/7, we don't need those targets.

Reply:
Removed NetMinimumSupportedVersion